src/web/app/pages-static/features-page/features-page.component.scss

```diff
@@ -2,3 +2,7 @@ img {
   margin: 15px 0;
   max-width: 80%;
 }
+
+.inner {
```

Comment:
Nitpicking: But could use a more descriptive classname here

Reply:
Make sense. Will make it more meaningful.